app.js

```diff
@@ -14,10 +15,36 @@ app.use(express.static(path.join(__dirname, 'public')));
 
 // Register the location for handlebars partials here:
 
-// ...
-
 // Add the route handlers here:
+hbs.registerPartials(path.join(__dirname, 'views/partials'));
 
+// Iteration 2_route endpoint home
+app.get('/', (req, res) => {
```

Comment:
is there a reason why you have the index route twice? also, did it not work without adding the .hbs suffix to your renders? it should read it automatically